packages/ra-ui-materialui/src/list/SingleFieldList.tsx

```diff
@@ -1,22 +1,18 @@
 import * as React from 'react';
-import { LinearProgress, styled, SxProps } from '@mui/material';
-import {
-    cloneElement,
-    Children,
-    HtmlHTMLAttributes,
-    ComponentType,
-} from 'react';
+import { Chip, Stack, StackProps, styled } from '@mui/material';
 import PropTypes from 'prop-types';
 import {
     sanitizeListRestProps,
     useListContext,
     useResourceContext,
     RaRecord,
     RecordContextProvider,
+    RecordRepresentation,
     ComponentPropType,
     useCreatePath,
 } from 'ra-core';
 
+import { LinearProgress } from '../layout/LinearProgress';
 import { Link } from '../Link';
 
 /**
```

Comment:
Since the component works without children, you might add an example to demonstrate it.